PostBoard.Api/Controllers/BirthdayController.cs

```diff
@@ -10,19 +11,26 @@ namespace PostBoard.Api.Controllers;
 [ApiController]
 public class BirthdayController : ControllerBase
 {
-    private static List<Birthday> Birthdays = new List<Birthday>();
+    //private static List<Birthday> Birthdays = new List<Birthday>();
+    private readonly PostBoardContext _context;
```

Comment:
Rename `_context` to `_dbContext`, so we know that it's exactly context of Database provider.